src/Indicator.vala

```diff
@@ -197,6 +202,17 @@ public class Power.Indicator : Wingpanel.Indicator {
         string? primary_text = null;
         string? secondary_text = null;
         if (display_device != null) {
+            if (display_device.percentage == LOW_BATTERY_PERCENTAGE && !display_device.is_charging) {
+                is_showing_low_battery_percent = true;
```

Comment:
You do not really need this variable.  Hiding the percentage when it is already hidden does no harm.